manifests/profile/solr.pp

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2018-2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::solr
+#
+# Provisions a basic instance of solr. Cores and specific implementation
```

Comment:
Remove the second sentence here -- that cores are elsewhere is implicit and the "specific implementation" bit is fluff.

I would just say: `Install Solr with base configuration.`